cmd/minikube/cmd/config/validations.go

```diff
@@ -50,6 +51,14 @@ func IsValidDiskSize(name string, disksize string) error {
 	return nil
 }
 
+func IsValidURL(name string, location string) error {
+	_, err := url.Parse(location)
```

Comment:
Yes - It actually allows pretty much anything...

https://play.golang.org/p/Vo3OLIdluJ